docs/spec/spec.md

```diff
@@ -0,0 +1,309 @@
+# Object Model
+
+This is the detailed specification of the Knative Eventing Object Model. For
+context, see the [overview](overview.md), [motivation](motivation.md), and the
+[interface contract](interfaces.md) documents.
+
+ * [Source](#kind-source) 
+ * [Channel](#kind-channel)
+ * [Subscription](#kind-subscription)
+ * [Provider](#kind-provisioner)
+
+---
+
+## kind: Source
+
+### group: eventing.knative.dev/v1alpha1
+
+_Describes a specific configuration (credentials, etc) of a source system which
+can be used to supply events. Sources emit events using a channel specified in
+their status. They cannot receive events._
+
+### Object Schema
+
+#### Spec
+
+| Field | Type | Description | Limitations |
+| --- | --- | --- | --- |
+| provisioner*| ProvisionerReference | The provisioner used to create any backing resources and configuration. | Immutable. |
+| arguments | runtime.RawExtension (JSON object)| Arguments passed to the provisioner for this specific source. | Arguments must validate against provisioner's parameters. |
+| channel | ObjectRef | Specify an existing channel to use to emit events. If empty, create a new Channel using the cluster/namespace default. | Source will not emit events until channel exists. |
+
+\*: Required
```

Comment:
I'd prefer bold for **required**.

Reply:
And italic for _oneof_.

I think you could put this into the top section with ObjectMeta info.

Reply:
I like this way less than subscripts.